src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/FieldBlocks.js

```diff
@@ -97,15 +100,32 @@ class FieldBlocks extends React.Component<FieldTypeProps<Array<BlockEntry>>> {
             return {};
         }
 
-        return Object.keys(settingsSchema).reduce((iconsMapping, schemaKey) => {
-            const blockSettingsTag = settingsSchema[schemaKey].tags.find((tag) => tag.name === SETTINGS_TAG);
+        const iconMappingReducerCreator = (prefixSchemaKey: string = '') => (iconsMapping, schemaKey) => {
+            const pointer = '/' + prefixSchemaKey + schemaKey;
+
+            if (!jsonpointer.has(settingsSchema, pointer)) {
+                return iconsMapping;
+            }
+
+            const schemaEntry = jsonpointer.get(settingsSchema, pointer);
+
+            if (!schemaEntry.tags && schemaEntry.items) {
```

Comment:
cant a `schemaEntry` have an unrelated tag but also further items?

Reply:
This code is there for `section`s, which never have a `tag`, but I get that this is a bit confusing... I am not sure anymore, but if I have another look I think the `items` property only exists on `sections` anyway, so I should be able to just check for its existence... E.g. something like:

```javascript
if (schemaEntry.items) {
    // ...
}
```

I'll try it, and see what happens.

Reply:
makes sense 🙂